authbridge/authlib/listener/forwardproxy/gzip_sse_test.go

```
@@ -0,0 +1,190 @@
+		t.Fatalf("pipeline.New: %v", err)
+	}
+	srv, err := NewServer(pipeline.NewHolder(pipe), store, nil)
+	if err != nil {
+		t.Fatalf("NewServer: %v", err)
+	}
+	proxy := httptest.NewServer(srv.Handler())
+	defer proxy.Close()
+
+	body := `{"model":"claude-sonnet-5","stream":true,` +
+		`"messages":[{"role":"user","content":"hi"}]}`
+	req, err := http.NewRequest("POST", upstream.URL+"/v1/messages", bytes.NewReader([]byte(body)))
```

golangci-lint (2.13.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)